src/App.js

```diff
@@ -37,7 +37,12 @@ const tweetsArray = [
 function App() {
   return (
```

Comment:
Una ventaja de usar arrow function es que tenemos el return implicito, entonces podemos refactorizar de la siguente manera y pasamos la key al componente mas 'padre' dentro de nuestro map 😉 
```suggestion
return (
    <div className="App">
      {tweetsArray.map((tw, index) => (
        <Tweet key={index} tweet={tw} />
      ))}
    </div>
  );```